app/assets/stylesheets/components/_tags.scss

```diff
@@ -1,3 +1,73 @@
+// Tags
+
+ul.tags {
+  display: block;
+  height: 2em;
+  margin: 0;
+  padding: 0;
+
+  & > li {
+    background: $grey;
+    border: 1px solid $grey;
+    border-radius: $radius;
+    color: $white;
+    cursor: pointer;
+    display: block;
+    font-size: 0.9em;
```

Comment:
Properties should be ordered background, border, border-radius, color, cursor, display, float, font-size, list-style, margin-right, padding, position, text-transform, transition